### PR TITLE
moved stuff into more appropriate sections

### PR DESCRIPTION
**Pull request by tonijarjour**
Just moved stuff into their relevant sections.

### CODE REVIEW

File: picom.sample.conf [hunk 1]
```diff
@@ -132,6 +138,69 @@ focus-exclude = [ "class_g = 'Cairo-clock'" ];
...
+# Try to detect WM windows (a non-override-redirect window with no 
+# child that has 'WM_STATE') and mark them as active.
+#
+# mark-wmwin-focused = false
+mark-wmwin-focused = true;
+
+# Mark override-redirect windows that doesn't have a child window with 'WM_STATE' focused.
+# mark-ovredir-focused = false
+mark-ovredir-focused = true;
+
```
Comment (yshui) [inline picom.sample.conf:150]: This doesn't belong here. focused state changes more than opacity.

File: picom.sample.conf [hunk 2]
```diff
@@ -132,6 +138,69 @@ focus-exclude = [ "class_g = 'Cairo-clock'" ];
...
+# not passing '_NET_WM_OPACITY' of client windows to frame windows.
+#
+# detect-client-opacity = false
+detect-client-opacity = true;
+
+# Use EWMH '_NET_ACTIVE_WINDOW' to determine currently focused window, 
+# rather than listening to 'FocusIn'/'FocusOut' event. Might have more accuracy, 
+# provided that the WM supports it.
+#
+# use-ewmh-active-win = false
```
Comment (yshui) [inline picom.sample.conf:161]: Doesn't belong here, same as the focused ones.

File: picom.sample.conf [hunk 3]
```diff
@@ -132,6 +138,69 @@ focus-exclude = [ "class_g = 'Cairo-clock'" ];
...
+# when redirecting/unredirecting windows.
+#
+# unredir-if-possible = false
+
+# Delay before unredirecting the window, in milliseconds. Defaults to 0.
+# unredir-if-possible-delay = 0
+
+# Conditions of windows that shouldn't be considered full-screen for unredirecting screen.
+# unredir-if-possible-exclude = []
+
```
Comment (yshui) [inline picom.sample.conf:174]: Doesn't belong here at all.

File: picom.sample.conf [hunk 4]
```diff
@@ -132,6 +138,69 @@ focus-exclude = [ "class_g = 'Cairo-clock'" ];
...
+# detect-transient is enabled, too.
+#
+# detect-client-leader = false
+detect-client-leader = true
+
+# Do not use EWMH to detect fullscreen windows. 
+# Reverts to checking if a window is fullscreen based only on its size and coordinates.
+#
+# no-ewmh-fullscreen = false
+
```
Comment (yshui) [inline picom.sample.conf:192]: Doesn't belong here, same as the focused ones

File: picom.sample.conf [hunk 5]
```diff
@@ -132,6 +138,69 @@ focus-exclude = [ "class_g = 'Cairo-clock'" ];
...
+# Dimming bright windows so their brightness doesn't exceed this set value. 
+# Brightness of a window is estimated by averaging all pixels in the window, 
+# so this could comes with a performance hit. 
+# Setting this to 1.0 disables this behaviour. Requires --use-damage to be disabled. (default: 1.0)
+#
+# max-brightness = 1.0
```
Comment (yshui) [inline picom.sample.conf:198]: This is for dimming, not transparency related

File: picom.sample.conf [hunk 6]
```diff
@@ -61,6 +61,12 @@ shadow-exclude = [
 # Crop shadow of a window fully on a particular Xinerama screen to the screen.
 # xinerama-shadow-crop = false
 
+# Try to detect windows with rounded corners and don't consider them 
+# shaped windows. The accuracy is not very high, unfortunately.
+#
+# detect-rounded-corners = false
+detect-rounded-corners = true;
```
Comment (yshui) [inline picom.sample.conf:68]: rounded_corners isn't shadow specific, it can be matched against in any type of rules.

**Review by yshui — CHANGES_REQUESTED** [review]

**Comment by tonijarjour** [comment]
Alright ty

**Pull request closed without merging**